lib/generators/rolemodel/css/icons/templates/config/webpack/loaders/svg-color-override-loader.js

```diff
@@ -33,7 +33,17 @@ module.exports = function(content) {
         name: 'removeViewBox',
         active: false
       },
-      'removeDimensions'
+      'removeDimensions',
+      {
+        name: 'prefixIds',
+        params: {
+          delim: '-',
+          prefix: () => {
+            const pathParts = this.resourcePath.split('/')
+            return pathParts[pathParts.length - 1].replace('.svg', '')
```

Comment:
Since this runs in Node, you could use [`path.basename()`](https://nodejs.org/dist/latest-v18.x/docs/api/path.html#pathbasenamepath-suffix).